packages/core/src/collections/notes.ts

```diff
@@ -199,7 +199,11 @@ export class Notes implements ICollection {
 
   get all() {
     return this.collection.createFilter<Note>(
-      (qb) => qb.where(isFalse("dateDeleted")).where(isFalse("deleted")),
+      (qb) =>
+        qb
+          .where(isFalse("dateDeleted"))
+          .where(isFalse("deleted"))
+          .where(isFalse("archived")),
       this.db.options?.batchSize
```

Comment:
I'm filtering out archived notes from the get all notes method. Should we also do the same for getting favorite notes? @thecodrr

Reply:
Yes. Archived notes will only be visible in the archived list. They won't show in notebooks, tags etc.